solver/src/http_solver.rs

```diff
@@ -46,7 +47,8 @@ impl SolverConfig {
                 "max_nr_exec_orders",
                 self.max_nr_exec_orders.to_string().as_str(),
             )
-            .append_pair("time_limit", self.time_limit.to_string().as_str());
+            .append_pair("time_limit", self.time_limit.to_string().as_str())
+            .append_pair("instance_name", self.instance_name.as_str());
```

Comment:
Why does the driver generate this? Is it logged somewhere here so that it can be cross-referenced?

Reply:
The driver generates this because the solver doesn't know have the info to properly name the instance (it doesn't know about networks and chains). Logging is a [good idea](https://github.com/gnosis/gp-v2-services/pull/701).

Reply:
I see, chain and network id are equivalent (or at least uniquely identify the network) so just passing a header with this could have been enough. But if we log it in the driver, it should be simple to find the exact file name in the AWS bucket when debugging a specific instance.

Reply:
It's bit confusing to me. By [this table](https://github.com/gnosis/gp-v2-services/blob/main/shared/src/network.rs) for example 

```
(network_id, chain_id) => network_name
-------------------------------------------------
("1", 1) => "Ethereum", 
("1", 101) => "EtherInc",
 ("3", 3) => "Ethereum",
```
from which I inferred that

* network_id != chain_id
* chain_id might encode everything (but is not user friendly)
* network_name perhaps could be improved in the table

So I decided to just include chain_id and network_name.